modules/govuk_containers/manifests/terraboard.pp

```diff
@@ -122,6 +122,7 @@
   }
 
   @@icinga::check { "check_terraboard_running_${::hostname}":
+    ensure              => absent,
     check_command       => 'check_nrpe!check_proc_running!terraboard',
```

Comment:
I think this should work, there's one for `gemstash` which seems to work: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk_containers/manifests/gemstash.pp#L38

https://alert.publishing.service.gov.uk/cgi-bin/icinga/extinfo.cgi?type=2&host=apt-1.management.publishing.service.gov.uk&service=gemstash+running